docs/getting-started/running.md

```diff
@@ -47,9 +47,12 @@ You can now create your [first flow](first-flow).
       -u, --userDir  DIR   use specified user directory
       -v                   enable verbose output
       -?, --help           show usage
+      [flows.json]         name of the data file you want to work with. Using different names, such as greenhouse.json
```

Comment:
As these lines are pre formatted the length matters.... 
![image](https://user-images.githubusercontent.com/5375409/42444415-f377f614-8367-11e8-85c0-b369a6e751f0.png)